x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings/latest_findings_container.tsx

```diff
@@ -42,11 +45,9 @@ export const getDefaultQuery = ({
   filters,
   sort: { field: '@timestamp', direction: 'desc' },
   pageIndex: 0,
-  pageSize: 10,
+  pageSize: MAX_ITEMS,
```

Comment:
this should be the default page display number, not `MAX_ITEMS`, which is like the default (and only) fetching number.

when there is no local storage value, as in, initial value comes from `getDefaultQuery`, we show `500` items on a single page: 

![Screen Shot 2022-11-14 at 21 10 12](https://user-images.githubusercontent.com/20814186/201745331-f849e8af-c40d-4380-8993-e9236ff1838a.png)

Reply:
I did a little refactoring around this. Removed pageSize from urlQuery, so we don't have two sources of truth.